presto-main/src/main/java/com/facebook/presto/connector/system/QuerySystemTable.java

```diff
@@ -129,6 +142,34 @@ private static Block resourceGroupIdToBlock(ResourceGroupId resourceGroupId)
         return blockBuilder.build();
     }
 
+    private Block sessionPropertiesToBlock(Map<String, String> properties)
+    {
+        // Since SessionRepresentation is only created via Session systemProperties is non-null
```

Comment:
Why have this constraint be enforced by a comment? I would either checkNotNull or handle nulls instead

Reply:
Good point:) Will do.